### PR TITLE
Test inicial scene

### PR DESCRIPTION
**Pull request by RicNorris**

### CODE REVIEW

File: inicial_scene.tscn
Comment (DiceWithDeath) [inline inicial_scene.tscn:1]: This is the old script that didn't work, needs to go in bin

**Review by tvardero — CHANGES_REQUESTED** [review]
I see there are commits from PR #4
So I'll block this PR until 4th is merged

**Pull request closed without merging**
